operations/workplace/people/hr-cadences.md

```diff
@@ -10,15 +10,74 @@ Listening sessions are 1x1 meeting with your HR partner. The intent of the HR Li
 
 Feedback themes will help drive HR priorities, communication and/or clarification efforts and even coaching opportunities for managers and leaders.
 
-**Cadence** Your HR partner will schedule time with you via Google Calendar. The goal is to meet with every person at least once per quarter. This may change due to HR team bandwidth. However, every staff member is encouraged to schedule time with someone on HR if there is not an existing scheduled meeting or if there’s an urgent topic.
+**Cadence** 
+The goal is monthly for managers and quarterly for non-managers. Your HR partner will schedule time with you via Google Calendar. This may change due to HR team bandwidth. However, every staff member is encouraged to schedule time with someone on HR if there is not an existing scheduled meeting or if there’s an urgent topic.
 
 Manager listening sessions will happen once a month. In addition to the above topics, this is also an opportunity for people managers to unpack questions around people management challenges, concerns and/or best practices. This feedback will also help drive future Manager Training topics.
 
 ## Staff Enablement Survey
 
-Information on [Staff Enablement Cadence](https://handbook.mattermost.com/operations/operations/company-cadence#staff-enablement-survey) can be found here.
+Staff enablement data provides a benchmark understanding of how staff feel or don’t feel enabled in the workplace. This is also a predictor of satisfaction and business outcomes. Quantitative data is a starting point to gather qualitative data for customized action planning.
+
+Mattermost uses Gallup’s 12 Engagement questions to assess how enabled staff at Mattermost feel. The [Gallup](https://q12.gallup.com/Help/en-us/Answers/164468) research indicates 12 of the biggest indicators of high-performing teams in terms of critical outcomes such as productivity, profitability, safety and recognition. We use all 12 questions because each  topics asked about are deeply important to the Mattermost  staff experience. We value high trust over micromanagement and want to continuously improve how we enable all staff to bring their best self to work. The bi-annual engagement survey helps HR, MLT and Managers focus on high impact initiatives that mean the most to the staff experience. 
```

Comment:
```suggestion
Mattermost uses Gallup’s 12 Engagement questions to assess how enabled staff at Mattermost feel. The [Gallup](https://q12.gallup.com/Help/en-us/Answers/164468) research indicates 12 of the biggest indicators of high-performing teams in terms of critical outcomes such as productivity, profitability, safety, and recognition. We use all 12 questions because each topic is deeply important to the Mattermost  staff experience. 

We value high trust over micromanagement and want to continuously improve how we enable all staff to bring their best self to work. The bi-annual engagement survey helps HR, MLT, and Managers focus on high impact initiatives that mean the most to the staff experience. 
```